config/kernel-monitor.json

```diff
@@ -14,16 +22,19 @@
 		},
 		{
 			"type": "permanent",
+			"condition": "KernelDeadlock",
 			"reason": "AUFSUmountHung",
 			"pattern": "task umount\\.aufs:\\w+ blocked for more than \\w+ seconds\\."
 		},
 		{
 			"type": "permanent",
+			"condition": "KernelDeadlock",
 			"reason": "DockerHung",
 			"pattern": "task docker:\\w+ blocked for more than \\w+ seconds\\."
 		},
 		{
 			"type": "permanent",
+			"condition": "KernelDeadlock",
 			"reason": "KernelBug",
```

Comment:
KernelBug is too generic here since above two can be in this category too. Can we change this to UnregisterNetDeviceIssue?